### PR TITLE
feat: revamp homepage

### PR DESCRIPTION
**Pull request by urmauur**
## Describe Your Changes

Screenshot 2024-03-14 at 17 23 50

Screenshot 2024-03-14 at 17 24 32

## Fixes Issues

- #2219 

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: docs/src/pages/index.js [hunk 1]
```diff
@@ -35,306 +40,461 @@ export default function Home() {
...
-                src="/img/homepage/element-hero-blur.webp"
-                alt="Element blur"
-                className="hidden lg:block absolute blur-3xl opacity-30 right-32 -bottom-32"
+          {/* Hero */}
+          <div className="text-center py-16">
+            <h1 className="text-5xl lg:text-8xl !font-normal leading-tight lg:leading-tight mt-2 font-serif">
+              Rethink the Computer
+            </h1>
+            <p className="text-2xl -mt-1 leading-relaxed text-black/60 dark:text-white/60">
+              Turn your computer into a{' '}
```
Comment (0xSage) [inline docs/src/pages/index.js:49]: "into an" grammar check

File: docs/src/pages/index.js [hunk 2]
```diff
@@ -35,306 +40,461 @@ export default function Home() {
...
+                </div>
+              ) : (
+                <Dropdown />
+              )}
+            </div>
+            <p className="mt-6 text-black/60 dark:text-white/60">
+              <span className="text-blue-500 font-bold dark:text-blue-500">
+                400K+
+              </span>{' '}
+              downloads | Free & Open Source
```
Comment (0xSage) [inline docs/src/pages/index.js:70]: Capitalize "Downloads"

File: docs/src/pages/index.js [hunk 3]
```diff
@@ -35,306 +40,461 @@ export default function Home() {
...
+                >
+                  <span>Download Jan for PC</span>
+                </div>
+              ) : (
+                <Dropdown />
+              )}
+            </div>
+            <p className="mt-6 text-black/60 dark:text-white/60">
+              <span className="text-blue-500 font-bold dark:text-blue-500">
+                400K+
```
Comment (0xSage) [inline docs/src/pages/index.js:68]: Remove the blue/bold style, i thought it was a link. No need to stylize this. Let's not link it to anything.

**Review by 0xSage — CHANGES_REQUESTED** [review]

**Comment by 0xSage** [comment]
image
bolding seems off

**Review by hieu-jan — APPROVED** [review]
LGTM!

**Comment by hieu-jan** [comment]
All request changed are solved. Ready to merge!

**Pull request merged**